From 770c8dcd50831138520ee689520ac1c62763fef7 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Wed, 25 Jul 2018 16:13:38 +0800
Subject: [PATCH 198/205] soc: ti: wkup_m3_ipc: give the proper info about
 scale data

This patch is to printk the scale data information if success since customer
might be not aware of this Scale Data has been applied into the system.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/soc/ti/wkup_m3_ipc.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/ti/wkup_m3_ipc.c b/drivers/soc/ti/wkup_m3_ipc.c
index 6099590..e1283c6 100644
--- a/drivers/soc/ti/wkup_m3_ipc.c
+++ b/drivers/soc/ti/wkup_m3_ipc.c
@@ -142,6 +142,7 @@ static void wkup_m3_scale_data_fw_cb(const struct firmware *fw, void *context)
 	val |= ((aux_base + hdr.wake_offset) << 16);
 
 	m3_ipc->volt_scale_offsets = val;
+	dev_info(dev, "PM: Voltage Scale Data binary installed.\n");
 
 release_sd_fw:
 	release_firmware(fw);
-- 
1.7.9.5

